nextcord/role.py

```diff
@@ -178,6 +179,10 @@ class Role(Hashable):
         Indicates if the role can be mentioned by users.
     tags: Optional[:class:`RoleTags`]
         The role tags associated with this role.
+    flags: :class:`RoleFlags`
```

Comment:
This shouldn't be in this section of the documentation since `flags` is a property now.